localization/replace/english/dlc/fp3/wc_dlc_fp3_culture_l_english.yml

```
@@ -0,0 +1,8 @@
+﻿l_english:
+ ###################################
+ # Olde Traditions
+ ###################################
+ tradition_fp3_beacon_of_learning_desc: "A nexus of scholarship, this culture prides itself on scientific advancement."
```

maybe we should make the tradition less externally focused, because some of the cultures that have it do not care much for outsiders or are cut off from the world.
```suggestion
 tradition_fp3_beacon_of_learning_name: "Academic Methodologies"
 tradition_fp3_beacon_of_learning_desc: "Driven by a desire for understanding of magic and the physical world, this culture prides itself on scientific advancement."
```

---

I think vanilla descriptions should be changed only when they make implicit reference to our real world. Which is why I think it's fine as it is right now, but I should find and equivalent for Jirga.

---

It kind of does make implicit reference to our real world, because in the real world, science is a characteristic of societies that are open and cosmopolitan. In Warcraft, science (before the First War) was a characteristic of societies that were exclusive, or at least that's not necessarily true. When you think about it, a lot of the events in warcraft boil down to scientific societies living or dying by accepting outsiders and the change they bring. Gnomes and Dwarves were solidly pro-alliance and thrived. High Elves were wishy washy and ended up going through a number of changes until their magic addiction opened them up to the world by force and they renamed themselves Blood Elves. Nightborne were closed off from the rest of the world for 10,000 years. There was an entire subplot of WOTLK with Dalaran adapting to changes in their social order, reckoning their closed and scholastic society with the horde and alliance visitors. So I think it's obvious magic and science in this world don't work like science in ours.

---

at the very least the trait should be changed for the Nightborne ethnicity. it's not a nexus of anything, it was a closed off city of people having a big dinner party for 10,000 years

---

like, i get philosopher culture because you can definitely talk philosophy at a dinner party, but you really can't say they were academically oriented on the same level Dalaran was attracting foreign scholars and mages.